src/Components/card/card.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Spoiler from "../spoiler/spoiler.js";
+import "./card.css";
+import data from "./data.js";
+import girlPicture from "./illustration-woman-online-desktop.svg";
+import bgPicture from "./bg-pattern-desktop.svg";
+import cubePicture from "./illustration-box-desktop.svg";
+
+const Card = () => {
+    const elements = data.map( (item) => {
+        if ( typeof item === 'object' && isEmpty(item) ){ 
```

Comment:
Do we really need this check?

Reply:
Also the condition is wrong, should be `typeof item === 'object' && !isEmpty(item)`